Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM mcr.microsoft.com/dotnet/sdk as build
+COPY . ./src
+WORKDIR /src
+RUN dotnet build "SzczurTv/SzczurTv.sln" -o /app
+RUN dotnet publish "SzczurTv/SzczurTv.sln" -o /publish
```

Comment:
we should dockerize the development version first, we don't need the production right now, we can add it later. this way we won't be blocked with development